app/scripts/modules/titus/src/instance/details/instanceDetails.html

```diff
@@ -145,6 +145,15 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
             ng-repeat="targetGroup in metric.targetGroups"
           >
             <instance-load-balancer-health load-balancer="targetGroup"></instance-load-balancer-health>
+            <span class="pad-left small">
```

Comment:
Would it be possible to move this markup into the `instanceLoadBalancerHealth` component?

Reply:
Yes definitely possible. This component is also used for LoadBalancer type health metrics, so I will hide this segment if the url is not present.